frontend/src/formdescriptioneditor/__tests__/__snapshots__/FormDescriptionEditorWebSocketContainer.test.tsx.snap

```diff
@@ -0,0 +1,69 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should render the FormDescriptionEditorWebSocketContainer 1`] = `
```

Comment:
Working with snapshots will just create way more problems than it will solve. It does not test anything here expect the rendering of the component which is not very helpful and this file would have to be massively updated each time the component will change and it will change a lot.

Consider adding `data-testid` attributes and rely on them instead

Reply:
Done